idaes/models_extra/power_generation/unit_models/boiler_heat_exchanger_2D.py

```diff
@@ -3165,7 +3171,12 @@ def initialize_build(
             blk.head_hin_eqn.deactivate()
 
         with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
-            res = opt.solve(blk, tee=slc.tee)
+            res = opt.solve(
+                blk,
+                tee=slc.tee,
+                options={"halt_on_ampl_error": "yes"},
```

Comment:
Should we really have these options in a working flowsheet (that we hope will be an example for users)?

Reply:
Oops, left over from debugging.